lib/internal/event_target.js

```diff
@@ -463,6 +474,14 @@ class NodeEventTarget extends EventTarget {
     this.addEventListener(type, listener, { [kIsNodeStyleListener]: true });
     return this;
   }
+  emit(type, arg) {
+    if (typeof type !== 'string') {
```

Comment:
Minor suggestion: use `validateString` from `/internal/validators`.
```suggestion
	validateString(type, 'type')
```